network-api/networkapi/settings.py

```diff
@@ -487,3 +489,9 @@ def show_toolbar(request):
 
 # Used by load_fake_data to ensure we have petitions that actually work
 PETITION_TEST_CAMPAIGN_ID = env('PETITION_TEST_CAMPAIGN_ID')
+
+# Buyers Guide Rate Limit Setting
+BUYERS_GUIDE_VOTE_RATE_LIMIT = env('BUYERS_GUIDE_VOTE_RATE_LIMIT')
+
+# Detect if we're testing
+TESTING = sys.argv[1:2] == ['test']
```

Comment:
this feels like it's position based, probably a good idea to harden this by doing an `in` check, like `TESTING = 'test' in sys.argv`?